java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1275,6 +1278,47 @@ public void zrank(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void bzmpop(BaseClient client) {
+        assumeTrue(REDIS_VERSION.isGreaterThanOrEqualTo("7.0.0"), "This feature added in redis 7");
+        String key1 = "{bzmpop}-1-" + UUID.randomUUID();
+        String key2 = "{bzmpop}-2-" + UUID.randomUUID();
+        String key3 = "{bzmpop}-3-" + UUID.randomUUID();
+
+        assertEquals(2, client.zadd(key1, Map.of("a1", 1., "b1", 2.)).get());
+        assertEquals(2, client.zadd(key2, Map.of("a2", .1, "b2", .2)).get());
+
+        assertArrayEquals(
+                new Object[] {key1, Map.of("b1", 2.)},
+                client.bzmpop(0.1, new String[] {key1, key2}, MAX).get());
+        assertArrayEquals(
```

Comment:
These should be tested with the correct order after Jonathan's LinkedHashMap PR gets merged, if not seems a bit pointless to test via max and min if set.length < count as we can't preserve order. So you probably should test where set.length > count to actually test the functionality.

Reply:
Good point, will keep this unresolved until then

Reply:
Added